XenAdmin/Wizards/BootModesControl.cs

```diff
@@ -135,6 +148,34 @@ private void UpdateControls()
             ShowTemplateWarning();
         }
 
+        private void UpdateTpmControls()
+        {
+            if (_templateVM != null)
+            {
+                var vtpmSupported = !Helpers.FeatureForbidden(_connection, Host.RestrictVtpm) &&
+                                    Helpers.XapiEqualOrGreater_vtpmtag(_connection) &&
+                                    (radioButtonUEFIBoot.Visible || radioButtonUEFISecureBoot.Visible) &&
+                                    (radioButtonUEFIBoot.Enabled || radioButtonUEFISecureBoot.Enabled);
+
+                groupBoxDevSecurity.Visible = vtpmSupported;
+                groupBoxDevSecurity.Enabled = vtpmSupported && !radioButtonBIOSBoot.Checked && !IsVtpmTemplate && _templateVM.VTPMs.Count < VM.MAX_ALLOWED_VTPMS;

```

Comment:
given the above && conditions are separated by line we should be consistent with these

Reply:
I just separated the ones above because they were longer and less readable, the conditions in line 160-161 are mostly single words - but indeed the line is too long, so I'll meet you halfway and split the lines in two.